lib/foreman_ansible_core/runner/ansible_runner.rb

```diff
@@ -41,7 +41,7 @@ def handle_event_file(event_file)
         logger.debug("[foreman_ansible] - parsing event file #{event_file}")
         begin
           event = JSON.parse(File.read(event_file))
-          if (hostname = event['event_data']['host'])
+          if (hostname = event.dig('event_data', 'host'))
```

Comment:
this is added in 2.3, do we have such ruby everywhere when this code runs?

Reply:
I think so, It only runs in smart proxy dynflow core, which runs in SCL and on debs there is ruby 2.5 iirc